src/renderer/styles/sidebar.less

```diff
@@ -3,7 +3,7 @@
 
 .Sidebar {
   background: @light-gray1;
-  overflow-y: auto;
+  overflow-y: scroll;
```

Comment:
This will have the unfortunate side effect of making the scrollbar permanently visible on Windows even if it doesn't need to scroll.  `auto` should, well, figure this out automatically.  Maybe try `overflow: auto` instead of just `overflow-y` 🤔

Reply:
that works  ¯\_(ツ)_/¯